general/app_releases.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Moodle App Release notes
+sidebar_label: Release notes
+tags:
+  - Moodle App
+  - Release notes
+---
+
+## Moodle App 4.x release notes
+
+| **Version name** | **Date** | **Version code** |
+|---|---|---|
+| [Moodle App 4.0.1](./v4/v4.0.1) | 26 May 2022 | 40100 |
+| [Moodle App 4.0.0](./v4/v4.0.0) | 22 April 2022 | 40000 |
+
+## Moodle App 3.x release notes
+
+Version 3 has been aligned with the Moodle LMS deployments.
+
+- From 3.0.0 to 3.4.2 it was based on Ionic 1 and supported Moodle 2.4 onwards.
+- On 3.5.0 name changed from Moodle Mobile to Moodle App. It was based on Ionic 3 and it was required to use Moodle 3.1 onwards.
+- On 3.9.5 the app was based on Ionic 5 version.
+
+| **Version name** | **Date** | **Version code** |
+|---|---|---|
+| [Moodle App 3.9.5](./v3/v3.9.5) | 27 August 2021 | 39500 |
```

Comment:
This needs to either be `./app_releases/v3/v3.9.5.md` _or_ this file needs to move to `./app_releases/index.md` as appropriate.

Reply:
Oh I'll check it ASAP Thanks for noticing. I've tried to move the file to index but somehow it didn't work, probably I was missing something. Anyway I'll review it.

Reply:
You have probably encountered the case where you cannot have an auto-generated directory with an index.md which you want to have as the _category_ page and not a member of the auto-generated sidebar.

Unfortunately that's a known limitation, and it's why the LMS release notes have the major version in the parent directory.

I do think that, ultimartely, the thing to do will be to have a `appVersions.json` file in the `data` directory with the below information, and to automatically generate this table, and the sidebar.

Reply:
Hi Andrew, I've changed the links. However about creating the json I would open a new issue since I have no time to do it right now. Do you mind?

Thanks!!!